scripts/ci/cache/__config.mjs

```diff
@@ -0,0 +1,174 @@
+// @ts-check
+import { resolve } from 'path'
+import { MOBILE_APP_DIR, ROOT } from './_common.mjs'
+import {
+  getPlatformString,
+  getYarnLockHash,
+  getPackageHash,
+  getNodeVersionString,
+  getGeneratedFileHash,
+  folderSizeIsEqualOrGreaterThan,
+  runCommand,
+  fileSizeIsEqualOrGreaterThan,
+  getPackageJSON,
+  getFilesHash,
+} from './_utils.mjs'
+
+import {
+  ENV_INIT_CACHE,
+  ENV_ENABLED_CACHE,
+  ENV_CACHE_SUCCESS,
+} from './_const.mjs'
+import { keyStorage } from './_key_storage.mjs'
+
+// When testing this is good to manipulate
+const HASH_VERSION = `newcache-${3}`
+
+export const ENABLED_MODULES = (process.env[ENV_ENABLED_CACHE] || '')
+  .split(',')
+  .map((x) => x.trim())
+  .filter((x) => x.length > 0)
+  .reduce((a, b) => {
+    a[b] = true
+    return a
+  }, {})
+
+export const cypressPath = '/github/home/.cache/Cypress'
+export const cacheSuccess = JSON.parse(process.env[ENV_CACHE_SUCCESS] ?? '{}')
+export const initCache = process.env[ENV_INIT_CACHE] === 'true'
+
+export const caches = [
+  {
+    enabled: ENABLED_MODULES['node_modules'],
```

Comment:
do we have to hardcode a new object if we decide to add a new directory to cache?

Reply:
no, path can also be an array

(which is what is supported by github actions cache)